cli/hse_cli.c

```diff
@@ -318,11 +319,11 @@ cmd_print_help(struct cli_cmd *cmd, FILE *fp)
     for (i = 0; nd[i].name; i++)
         width = max(width, strlen(nd[i].name));
     width += 4;
-    width = max(width, 24);
+    width = max(width, 24U);
     for (i = 0; nd[i].name; i++) {
         if (i == 0)
             fprintf(fp, "\n\n%*sOptions:\n", ilvl * tabw, "");
-        fprintf(fp, "%*s%-*s%s%s", (ilvl + 1) * tabw, "", width,
+        fprintf(fp, "%*s%-*s%s%s", (ilvl + 1) * tabw, "", (int)width,
```

Comment:
You have to pass an int here so downcasting/sign changes are unavoidable. Do you see a better way to do this?

Reply:
From a type/overflow standpoint this stuff was/is broken either way.  The changes here to width don't fix it, they just move the problem around.  I think they only way to truly fix it would be to capture the strlen() in a different variable, then abort if it plus width overflows INT_MAX, then add it to width..